### ISSUE TITLE
Error 'No module named datetime' when compiling shaderc library

### ISSUE THREAD

**Issue by fancycww** [android/ndk#1370]
#### Description
Hi,

I tried to build `shaderc` library for Vulcan development, run these commands:
```
cd ndk-bundle/sources/third_party/shaderc
../../../ndk-build.cmd NDK_PROJECT_PATH=. APP_BUILD_SCRIPT=Android.mk APP_ABI=all APP_PLATFORM=android-30 APP_STL=c++_static -j8 clean libshaderc_combined
```

But I got an error:
```
Traceback (most recent call last):
  File "third_party/spirv-tools/utils/update_build_version.py", line 32, in <module>
    import datetime
ImportError: No module named datetime
```
![image](https://user-images.githubusercontent.com/5031381/98774864-29794700-23ec-11eb-9777-87dc4c7236fb.png)

Any ideas on this?
Thanks.

#### Environment Details

* NDK Version: 21.3.6528147
* Build system: ndk-build
* Host OS: Windows 10
* ABI: all
* NDK API level: 24, 26, 30
* Device API level: 24, 26, 30
* Python: 3.8.3

**Comment by DanAlbert** [comment]
@hhb I think you recently fixed this for r22? Is this practical to fix for r21?

@fancycww FYI, the Vulkan team no longer recommends getting the validation layers from the NDK, and that you should instead get them from the Vulkan GitHub. See https://developer.android.com/ndk/guides/graphics/validation-layer?release=r21

**Comment by hhb** [comment]
This is the same as [issue 1291](https://github.com/android/ndk/issues/1291). It is fixed for r22 in [aosp/1351127](https://android-review.googlesource.com/c/platform/prebuilts/gcc/linux-x86/host/x86_64-w64-mingw32-4.8/+/1351127). We can cherry pick this change to 21e.

**Comment by fancycww** [comment]
Thanks.
